app/presenters/supergroups/policy_and_engagement.rb

```diff
@@ -60,5 +60,19 @@ def consultation_closing_date(base_path)
 
       date.strftime("Closing date %d %B %Y")
     end
+
+  private
+
+    def reorder_tagged_documents_to_prioritise_consultations
+      consultations = @content.select do |content_item|
+        consultation?(content_item.content_store_document_type)
+      end
+
+      other_document_types = @content.reject do |content_item|
+        consultation?(content_item.content_store_document_type)
+      end
```

Comment:
Would this be a shorter alternative?
```ruby
other_document_types = @content - consultations
```

Reply:
I'll give it a go! Thanks!

Reply:
@cbaines Updated and fixed up.

Reply:
If it works, then I think that's good :+1: